tests/integration/test_base_git.py

```diff
@@ -2,18 +2,41 @@
 
 from packit.actions import ActionName
 from packit.base_git import PackitRepositoryBase
-from packit.config import PackageConfig
+from packit.config import PackageConfig, Config
+
+# TODO rename it to
+# from sandcastle.api import Sandcastle
+# as https://github.com/packit-service/sandcastle/pull/9
+# is merged
+from generator.deploy_openshift_pod import OpenshiftDeployer
 
 
 def test_get_output_from_action_defined():
     echo_cmd = "echo 'hello world'"
 
     packit_repository_base = PackitRepositoryBase(
-        config=flexmock(),
+        config=flexmock(Config()),
         package_config=flexmock(PackageConfig(actions={ActionName.pre_sync: echo_cmd})),
     )
 
     packit_repository_base.local_project = flexmock(working_dir=".")
 
     result = packit_repository_base.get_output_from_action(ActionName.pre_sync)
     assert result == "hello world\n"
+
+
+def test_get_output_from_action_defined_in_sandcastle_object():
```

Comment:
This looks like a unit test given everything is mocked.

Reply:
The propose of this test is to be sure, that OpenshiftDeployer is really called and `run_command` is executed.
Nothing complicated.